login-workflow/src/new-architecure/components/workflow/Utility.ts

```diff
@@ -0,0 +1,26 @@
+import generateUtilityClass from '@mui/material/generateUtilityClass';
+
+export type WorkflowCardClasses = {
+    root?: string;
+    card?: string;
+};
+
+export type WorkflowCardClassKey = keyof WorkflowCardClasses;
+
+export function getWorkflowCardUtilityClass(slot: string): string {
```

Comment:
instead of defining a function like this for every component, let's use one function (getUtilityClasses) that takes two parameters (one for component name and one for the slot)